src/main/java/app/chatbot/Parser.java

```diff
@@ -0,0 +1,99 @@
+package app.chatbot;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import app.command.AddCommand;
+import app.command.Command;
+import app.command.CommandNotFoundException;
+import app.command.DeleteCommand;
+import app.command.ExitCommand;
+import app.command.FindCommand;
+import app.command.ListCommand;
+import app.command.MarkAsDoneCommand;
+import app.command.MarkAsUndoneCommand;
+import app.task.TaskTypes;
+
+
+/**
+ * Contains methods for parsing user input into the command line,
+ * and converting input into executable Commands.
+ */
+public class Parser {
+
+    /**
+     * Splits a String line of input into its command (the first word), and its subsequent arguments.
+     * By default, the key "Command" maps to the name of the command input (the first word).
+     * <br>
+     * The name of the Command is also a key to its default arguments;
+     * for eg: "delete 1" creates a mapping of "delete" -> "1".
+     * <br>
+     * In the case of adding a new event, the name of the command is mapped to its description;
+     * for eg: "event" -> "attend wedding"
+     *
+     * @param input arguments to be split
+     * @return map a mapping of each argument name to its value
+     */
+    private static Map<String, String> splitArgs(String input) {
+        // returns a Map mapping name of arg to arg value based on the raw format
+        // raw format according to task is /<name> <value>
+
+        Map<String, String> map = new HashMap<>();
+        String[] args = input.split(" /");
```

Comment:
Not really an issue to use `args` here but be careful if this was in the main class.